extensions/amp-sticky-ad/0.1/amp-sticky-ad.js

```diff
@@ -61,6 +61,8 @@ class AmpStickyAd extends AMP.BaseElement {
   layoutCallback() {
     // Reschedule layout for ad if layout sticky-ad again.
     if (this.visible_) {
+      toggle(this.element, true);
```

Comment:
Since sticky-ad display could be set to none when ad collapse, I want to make sure to toggle display to true when re layout